brave/src/test/java/zipkin2/reporter/brave/JsonV2RoundTripTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2016-2024 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.reporter.brave;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import zipkin2.codec.SpanBytesDecoder;
+
+class JsonV2RoundTripTest extends RoundTripTest {
+  JsonV2RoundTripTest() {
+    super(MutableSpanBytesEncoder.JSON_V2, SpanBytesDecoder.JSON_V2);
+  }
+
+  @Override @Disabled @Test void differentErrorTagName() {
+    // awaiting https://github.com/openzipkin/brave/pull/1415 to be released
```

Comment:
once this merges I'll cut brave 6.0.1 and that should scrub out this disabled thingy